src/content/apis/miscs/forem.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Forem API
+description: Forem API, popularly used for Dev.to.
+categories:
+  - miscs
```

Comment:
```suggestion
  - social
  - development
```

Reply:
I thought about development, but it's not exactly related to development. Though I agree it's more of social.

Reply:
Sure, sounds good to me